tests/test_cookies.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+
+from chocs.cookies import Cookie
+
+
+def test_can_instantiate():
+    headers = Cookie("key", "value")
+    assert isinstance(headers, Cookie)
+
+
+def test_cookie_as_string():
+    assert str(Cookie("key", "value", "test.com", "/", datetime(2020, 5, 17))) == (
+        "Set-Cookie: key=value; Domain=/; expires=Sun, 17 May 2020 00:00:00 GMT; "
+        "Path=test.com"
+    )
+
+
+def test_cookie_as_header():
```

Comment:
In function which return none please use `-> None` return type :)

Reply:
Even in tests? No other tests have return type annots.

Reply:
Sorry :P I should actually fix all the test when I look at it. Mainly the reasoning about having `return None` is to indicate that nothing should be returned form the function, like php's `void`